inspire_dojson/common/rules.py

```diff
@@ -53,6 +53,532 @@
+    'PRL': 'Phys. Rev. Lett. Server',
+    'PRLTA': 'Phys. Rev. Lett. Server',
+    'PRPLC': 'Physics Reports Server',
+    'PRSLA': 'Proceedings of the Royal Society Server',
+    'PRSTA': 'Phys.Rev.ST Accel.Beams Server',
+    'PTPKA': 'Prog.Theor.Phys. Server',
+    'PTRSA': 'Phil.Trans.Roy.Soc.Lond. Server',
+    'PUBMED': 'PUBMED Server',
+    'RCCHB': 'La Recherche Server',
+    'RESOF': 'Resonance Journal of Science Education Server',
+    'RINGBERG': 'Ringberg Conference Server',
+    'RJMPE': 'Russ. J. Math. Phys. Server',
+    'RMEAE': 'Radiation Measurements Server',
+    'RMHPB': 'Reports on Mathematical Physics Server',
+    'RMP': 'Rev. Mod. Phys. Server',
+    'RMPHA': 'Rev. Mod. Phys. Server',
+    'RMPHE': 'Reviews in Mathematical Physics Server',
+    'RMXFA': 'Rev. Mex. Fis. Server',
+    'RORPE': 'Rom. Rep. Phys. Server',
+    'RPDOD': 'Radiation Protection Dosimetry Server',
+    'RPPHA': 'Rep. Prog. Phys. Server',
+    'RRPQA': 'Rom. J. Phys. Server',
+    'RSI': 'Rev.Sci.Instrum. Server',
+    'RSINA': 'Rev. Sci. Instrum. Server',
+    'SACLAY': 'Saclay Document Server',
+    'SALAM': 'ICTP Preprint Archive',
+    'SCAMA': 'Scientific American Server',
+    'SCIDIR': 'Science Direct',
+    'SCIEA': 'Science Server',
+    'SCIENCEBLOG': 'Science Blog article',
+    'SCIENCEDAILY': 'Science Daily Article',
+    'SERVER': 'Electronic Version from a server',
+    'SERVER2': 'Electronic Version from another server',
+    'SLAC': 'SLAC Document Server',
+    'SLACBOOK': 'SLAC Book Catalog Record',
+    'SLACPUB': 'SLAC Document Server',
+    'SLACREPT': 'SLAC Document Server',
+    'SLACTN': 'SLAC Document Server',
+    'SLACTODAY': 'SLAC Today article',
+    'SOPJA': 'Russ.Phys.J. Server',
+    'SPACE': 'SPACE.com Article',
+    'SPIE': 'SPIE Server',
+    'SPRINGER': 'Springer Books Server',
+    'SPTPA': 'Tech. Phys. Server',
+    'SSC': 'HEPNET Document Server',
+    'SSI91': 'SLAC Document Server',
+    'SSI92': 'SLAC Document Server',
+    'SSI93': 'SLAC Document Server',
+    'SSI94': 'SLAC Document Server',
+    'SSI96': 'SLAC Document Server',
+    'SSI97': 'SLAC Document Server',
+    'SSI98': 'SLAC Document Server',
+    'TECHPHYS': 'Tech. Phys. Server',
+    'TECHPHYSL': 'Tech. Phys. Lett. Server',
+    'TJPHE': 'Turkish Journal of Physics Server',
+    'TMFZA': 'Teor.Mat.Fiz. Server',
+    'TMPHA': 'Theor.Math.Phys. Server',
+    'TORONTO-A': 'Toronto U. Astron. Thesis Server',
+    'TP': 'Tech. Phys. Server',
+    'UCOL': 'University Coll. London Server',
+    'UMI': 'UMI Thesis Server',
+    'VAC': 'Vacuum Document Server',
+    'VACUA': 'Vacuum Server',
+    'VIDEO': 'Watch the video of the talk',
+    'WORLDSCI': 'World Scientific Books Server',
+    'ZBLATT': 'Zentralblatt MATH Server',
+    'ZFDOF': 'Journal of Physical Studies Server',
+    'ZNTFA': 'Z. Naturforsch. Server',
+    'ZPA': 'Z. Phys. A Server',
+    'ZPC': 'Z. Phys. C Server',
+    'FERMILABR': 'Fermilab Library Server (fulltext available)',
+}
```

Comment:
Do we know if this is the latest KB available, or if updates were made to it on Legacy?

CC: @kaplun @tsgit

Reply:
@kaplun can you detail the one entry that has been added?

Reply:
['ECONFPDF---pdf from eConf']

the live kbr in db has 524 entries, the file in git 
https://github.com/inspirehep/inspire/blob/master/kbs/weblinks.kb
has 523 entries. the difference is the entry above

Reply:
```
In [2]: strings = []

In [3]: with open('kbs/weblinks.kb', 'r') as fh:
   ...:     for line in fh:
   ...:         strings.append[line.rstrip()]

In [5]: len(strings)
Out[5]: 523

In [6]: from invenio.bibknowledge import get_kbr_items

In [7]: livelist = []

In [8]: for x in get_kbr_items('WEBLINKS'):
   ...:     livelist.append("%s---%s" % (x['key'],x['value']))
   ...:     

In [9]: len(livelist)
524

In [10]: [s for s in livelist if s not in strings]
['ECONFPDF---pdf from eConf']

```

Reply:
Yeah exactly. Sorry, I have forgot to update the entry here.